src/sentry/migrations/0238_fill_org_onboarding_tasks.py

```diff
@@ -0,0 +1,581 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        # These are constants, not models
+        from sentry.models import OnboardingTask, OnboardingTaskStatus
+        from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+        Group = orm['sentry.Group']
+        Organization = orm['sentry.Organization']
+        OrganizationMember = orm['sentry.OrganizationMember']
+        OrganizationOnboardingTask = orm['sentry.OrganizationOnboardingTask']
+        Project = orm['sentry.Project']
+
+        queryset = Organization.objects.all()
+
+        for org in RangeQuerySetWrapperWithProgressBar(queryset):
+            if Project.objects.filter(organization=org).count() > 0:
+                OrganizationOnboardingTask.objects.create(
+                    organization=org,
+                    task=OrganizationOnboardingTask.FIRST_PROJECT,
+                    status=OnboardingTaskStatus.COMPLETE
+                )
+
+            projects = Project.objects.filter(organization=org).exclude(first_event=None).order_by('first_event')
+            project_count = projects.count()
```

Comment:
This is relatively expensive if there were a large number of projects.

Really what you're asking for here is "are there 0, 1, or 1+" projects. You don't care about the exact count.

In that case, you should add a LIMIT clause in here with `[0:2]`. This will apply `LIMIT 2`. Now you can continue the same comparison with a much cheaper `COUNT` query.